migrations/versions/3dc2df4235a7_adding_initial_tables.py

```diff
@@ -0,0 +1,72 @@
+"""Adding initial tables
+
+Revision ID: 3dc2df4235a7
+Revises: 
+Create Date: 2020-05-09 17:37:44.233485
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3dc2df4235a7'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('batches',
+    sa.Column('batchId', sa.Integer(), nullable=False),
+    sa.Column('createdAt', sa.DateTime(timezone=True), nullable=False),
+    sa.Column('publishedAt', sa.DateTime(timezone=True), nullable=True),
+    sa.Column('shiftLead', sa.String(length=100), nullable=True),
+    sa.Column('batchNote', sa.String(), nullable=True),
+    sa.Column('dataEntryType', sa.String(), nullable=True),
+    sa.Column('isPublished', sa.Boolean(), nullable=False),
+    sa.Column('isRevision', sa.Boolean(), nullable=False),
+    sa.PrimaryKeyConstraint('batchId')
+    )
+    op.create_table('states',
+    sa.Column('state', sa.String(), nullable=False),
+    sa.Column('fullName', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('state')
+    )
+    op.create_table('core_data',
```

Comment:
Do we really want to have the table names be  dunner case?  Seems like we should align them with either the Python class names (CoreData) or the json (coreData).

Reply:
renamed to coreData for JSON consistency, since we're already doing away with all Python conventions here in the column names (sadness)